src/locales/en.json

```diff
@@ -113,9 +113,10 @@
     "insurance": "Insurance",
     "isp": "ISP",
     "others": "Others",
+    "mes_infos": "MesInfos experimentation",
```

Comment:
Shouldn't this kind of category be present in the registry and not in a public app ? We are reproducing the same pattern as Collect with its `konnectors.json`

Reply:
Actually, the store defines a list of authorized categories so the locales should also be here. Unauthorized categoires will be changed to `other` then.

Reply:
IMHO, it is a role which should be delegated to the registry in the future.